USER-GUIDE.md

```diff
@@ -0,0 +1,484 @@
+# KCP-Edge User Guide:
+
+## Required Packages:
+   - docker
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Setup Instructions
+
+This guide is intended to explain how KCP-Edge works with a detailed example.
+
+### 1. Create your own edge infrastructure (edge clusters) - Bring Your Own Clusters (BYOC)
+
+Create your edge clusters or bring your own k8s edge clusters. In this example, we deploy only a single cluster using [kind](https://kind.sigs.k8s.io/). Our edge cluster is named “florin”:
+
+```shell
+kind create cluster --name florin
+```  
+
+### 2. Install KCP
+
+Since `KCP-Edge` leverages [`kcp`](kcp.io) logical workspace virtualization capability, we have to install `kcp` v0.11.0 binaries first by following the detailed installation instructions available [on the kcp website](https://docs.kcp.io/kcp/main/#download-kcp) or by using the script below:
```

Comment:
"logical workspace" mixes terminology; kcp has "logical clusters" and "workspaces".  I suggest to pick one of those.
Adding "virtualization" is confusing.  Logical clusters/workspaces _are_ virtualization (of the apiserver); adding the word "virtualization" suggests that we are talking about virtualization of the workspaces, which is not what we mean.

Reply:
Fixed